app/views/deploymentApproval/deploymentReview.scala.html

```diff
@@ -0,0 +1,64 @@
+@import model.APIApprovalSummary
+
+@(form: Form[controllers.HandleApprovalForm], apiDefinition: APIApprovalSummary)(implicit request: Request[_], userFullName: Option[String], flash: Flash, messages: Messages, appConfig: config.AppConfig)
```

Comment:
Could you add a unit test for the two new views, there should be a good example of how to do this.